pkg/operators/wasm/wasm.go

```diff
@@ -65,16 +66,37 @@ func (w *wasmOperator) InstantiateImageOperator(
 	operators.ImageOperatorInstance, error,
 ) {
 	instance := &wasmOperatorInstance{
-		gadgetCtx: gadgetCtx,
-		handleMap: map[uint32]any{},
-		logger:    gadgetCtx.Logger(),
+		gadgetCtx:   gadgetCtx,
+		handleMap:   map[uint32]any{},
+		logger:      gadgetCtx.Logger(),
+		paramValues: paramValues,
 	}
 
 	if err := instance.init(gadgetCtx, target, desc); err != nil {
 		instance.close(gadgetCtx)
 		return nil, fmt.Errorf("initializing wasm: %w", err)
 	}
 
+	cfg, ok := gadgetCtx.GetVar("config")
+	if !ok {
+		return nil, fmt.Errorf("missing configuration")
+	}
+	v, ok := cfg.(*viper.Viper)
+	if !ok {
+		return nil, fmt.Errorf("invalid configuration format")
+	}
+	instance.config = v
```

Comment:
Should we just ignore a non-existent config here? Otherwise we're introducing basically a hard dependency on the viper instance that is currently only served by the oci operator.

Reply:
You're right. Let's ignore it for now. Even if IMO the config is going to be more and more informant and perhaps we can make it a first class citizen later on.